lib/hyper_client.rb

```diff
@@ -226,11 +222,9 @@ def oauth_password_client
 
   def oauth_refreshtoken_client
     @oauth_refreshtoken_client ||= begin
-      @configuration.refresh  # Ensure we have the latest API credentials
-
       OAuth2::Client.new(nil,nil,
                          site: @configuration[:uc6_oauth_endpoint],
-                         connection_opts: @configuration[:uc6_proxy_host].present? ?
+                         connection_opts: !@configuration.blank_value?(:uc6_proxy_host) ?
```

Comment:
If you like you could use the same "present_value?" method that I mentioned before